TweetClassifier/src/main.py

```diff
@@ -16,7 +16,7 @@
 from nltk.corpus import stopwords
 
 # TODO: Change preprocess_data_train_test_split for binClass to work
-# allow printing full width in console
```

Comment:
This wasn't part of the todo, it was just a comment explaining the three lines underneath it.

Reply:
Fixed